README.md

```diff
@@ -72,28 +80,90 @@ This template follows a very simple project structure:
+The idea of this section is to explain how the template composition is better and easier to use when you try to use well formed arquitectures, especially using redux flow.
+
+First keep in mind that you will see the same way of classification in multiple places of the app. Different resources orchestrated by an index.js that allow you to call it as a single instance. This structure lets you use a simple way to import things.
+
+With that in mind we are going to look at each folder to explain how to use it.
+
+## Components
+
+The components are the basic structure of an application in react native, but specifically for this template the idea of ​​keeping the complexity of the development to a minimum, for this the ideal is that all the components are at the same nesting folder level. A component is only created if a specific view or functionality is being worked on or if the same object is to be reused in multiple parts of the application.
+
+Another important thing is the use of propTypes to check the kind of data that your components need to work properly. If the component receives some data from others, the type of this props must be defined and in case you need it the default value of the property too.
+
+### Static resources:
+
+To keep an application scalable and organized, the global static resources that are used in the application have to be created in a specific file.
+
+### We manage three main folders for that:
+
+- Assets: Here you can store all the images and icons that you need through the app. You have as example the icon ic_home.png, to respond with the different device screen densities just create inside the same folder the image and all the scaled versions that you need.
+- Localization: This folder contains all the language objects that you need to create a multilingual application. Create a file for each language, inside define an object and maintain the nesting sorted by the screen that contains the text that you need and the text you want to show. As the last step, remember to create a reference inside the Localization.js file and add to LocalizedStrings.
+- Theme: Here you can define all the styles that you use in different screens. To make easier the interaction of the application with device options as example you can create here assets as light and dark color palette
+
+## Redux
+
+Once the components are defined, they are tied to the management of information through the application. For this, Redux is implemented with the store-reducer-action structure as usual, however, not only the data is handled through the actions but the success and error responses are also defined by the same form.
+
+The template uses hooks for communication with the redux store. In that way it has two scenarios.
+
+- When you need to call an action you have to create an useDispatch function that lets you interact with the store.
+- To see what’s inside the store you should create a useSelector to extract specific info from the general store.
```

Comment:
I would remove this since it describes how to use react-redux. Although it could be a good hint for newcomers I. think we should aim to keep the README as simple as possible.